packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -19,6 +19,18 @@ contract Bridge is EssentialContract, IBridge {
     using LibAddress for address;
     using LibAddress for address payable;
 
+    /// @dev Gas overhead if a message is received but not processed in the same transaction.
+    // The measured value  is 104047
+    uint256 public constant GAS_RECEIVING = 110_000 + 21_000;
```

Comment:
Brecht, if I measure the gas used inside the functions, shall I add 21000 to it?